app/protocol/flows/v4/blockrelay/ibd_with_headers_proof.go

```diff
@@ -32,6 +33,8 @@ func (flow *handleIBDFlow) ibdWithHeadersProof(highHash *externalapi.DomainHash,
 		return err
 	}
 
+	log.Infof("Header download stage of IBD with pruning proof completed successfully from %s. "+
```

Comment:
Same here. Sometimes the consensus commit triggers a previous consensus delete with the line `Deleting database prefix &{0}`. We should explain first.